src/App.tsx

```diff
@@ -5,15 +5,20 @@ import routes from './router'
 
 import { CssBaseline } from '@mui/material'
 import ThemeProviderWrapper from './theme/ThemeProvider'
+import React, {useState} from 'react'
+import {UserContext} from './content/Auth/UserContext'
 
 const App = () => {
     const content = useRoutes(routes)
+    const [JWT, setJWT] = useState(null)
```

Comment:
It is good practice for each context to have its own provider, by creating an `AuthProvider`. Then this state can be moved into the provider.